### PR TITLE
Update lastest default version description on FAQ doc

### PR DESCRIPTION
**Pull request by lilyjazz**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by lilyjazz** [comment]
@qiancai @lucklove PTAL

**Review by lucklove — APPROVED** [review]

File: tidb-cloud/tidb-cloud-faq.md [hunk 1]
```diff
@@ -41,7 +41,9 @@ No.
 
 ### What versions of TiDB are supported on TiDB Cloud?
 
-For the currently supported TiDB version, see [TiDB Cloud Release Notes](/tidb-cloud/tidb-cloud-release-notes.md).
+Since June 20th, 2023, the latest default TiDB version for new TiDB Dedicated clusters is v6.5.3. In TiDB Serverless Cluster, it is v6.6.0.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-faq.md:44]:
```suggestion
- Starting from June 20, 2023, the default TiDB version for new TiDB Dedicated clusters is v6.5.3. 
- Starting from March 7, 2023, the default TiDB version for new TiDB Serverless clusters is v6.6.0.
```

File: tidb-cloud/tidb-cloud-faq.md [hunk 2]
```diff
@@ -41,7 +41,10 @@ No.
 
 ### What versions of TiDB are supported on TiDB Cloud?
 
-For the currently supported TiDB version, see [TiDB Cloud Release Notes](/tidb-cloud/tidb-cloud-release-notes.md).
+- Starting from June 20, 2023, the default TiDB version for new TiDB Dedicated clusters is v6.5.3. 
+- Starting from March 7, 2023, the default TiDB version for new TiDB Serverless clusters is v6.6.0.
+
+For the more information, see [TiDB Cloud Release Notes](/tidb-cloud/tidb-cloud-release-notes.md).
```
Comment (ran-huang) [inline tidb-cloud/tidb-cloud-faq.md:47]:
```suggestion
For more information, see [TiDB Cloud Release Notes](/tidb-cloud/tidb-cloud-release-notes.md).
```

**Review by ran-huang — APPROVED** [review]
rest LGTM

**Review by qiancai — APPROVED** [review]
/lgtm

**Pull request merged**